free-science-books.md

```diff
@@ -1,6 +1,25 @@
 **Note**: In order of [Library of Congress Classification](http://www.loc.gov/catdir/cpso/lcco/)
 
-# [(Q)](http://www.loc.gov/aba/cataloging/classification/lcco/lcco_q.pdf) Science (General)
+
+# [(B)](https://www.loc.gov/aba/cataloging/classification/lcco/lcco_b.pdf)  PHILOSOPHY, PSYCHOLOGY, RELIGION
+
+## (BF) Psychology
+
+* [Cognitive Science](http://www2.fiit.stuba.sk/~kvasnicka/CognitiveScience/Friedenberg_Cognitive%20science.pdf) - Jay Friedenberg and Gordon Silverman (PDF)
```

Comment:
Doesn't seem to be an authorized distribution. Do you have info otherwise?

Reply:
Ah I think you're right. We can scrap this one (I'm new to open source contributing and just noticed the `contributing.md` document. I'm realizing it explains a lot of the comments you're pointing to, sorry about that!